Include/Acidanthera/Library/OcBootManagementLib.h

```diff
@@ -2075,6 +2078,40 @@ OcAddEntriesFromBootEntryProtocol (
   IN     BOOLEAN        CreateForHotKey
   );
 
+#pragma pack (push, 1)
+
+/**
+  Useful internal vars from firmware UI protocol (cMP 144.0.0.0.0 ROM).
+**/
+typedef PACKED struct {
+  BOOLEAN                         mGopAlreadyConnected;
```

Comment:
This is really packed with pointers misaligned?

Reply:
Doh - sleep deprived when add this - after actually using those table pointers for debug early (when it wasn't marked packed, ofc). Thank you!

Reply:
Done in upcoming commit, so resolving.